HPCC-14219 Using Multiple Credentials with Nagios

g-pan opened the pull request:
Fix HPCC-14219 Using Multiple Credentials with Nagios
Add documentation for using multiple credentials for multiple ESPs.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>
@JamesDeFabia please review.

g-pan committed:
HPCC-14218 Using Multiple Credentials with Nagios

Fix HPCC-14218 Using Multiple Credentials with Nagios
Add documentation for using multiple credentials for multiple ESPs.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest
Sha: 7cb059b6ad5391f6d136095b6de81bbdd081cf17
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 90.752 seconds |

Install hpccsystems-platform-community_6.0.8-rc2.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|87 |86 |0 |1 |

Error(s):
CcdFileTest run timeout



HPCC Stop: OK
HPCC Uninstall: OK

JamesDeFabia commented on docs/HPCCMonitoring/HPCCMonitoringAndReporting.xml:
```diff
@@ -1191,6 +1190,17 @@
+
+                    <entry>Set specific logins credentials for esp checks. All
+                    fields are required. (esp name, user name, password) Can
+                    be specified more than once.</entry>
```
Set specific login credentials for ESP checks. All fields are required (esp name, user name, password). Can
be specified more than once to support multiple ESP servers.

g-pan committed:
HPCC-14219 Using Multiple Credentials with Nagios

Fix HPCC-14219 Using Multiple Credentials with Nagios
Add documentation for using multiple credentials for multiple ESPs.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

JamesDeFabia reviewed (approved):
Looks good to merge

HPCCSmoketest commented:
Automated Smoketest
Sha: b898cd7e62f3174173cf5b3c58d46da781fda651
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 90.894 seconds |

Install hpccsystems-platform-community_6.0.8-rc2.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|87 |86 |0 |1 |

Error(s):
CcdFileTest run timeout



HPCC Stop: OK
HPCC Uninstall: OK
